src/components/form/fields/FieldSetField.tsx

```diff
@@ -22,20 +22,32 @@ type FieldWithoutOptions = FieldBase & {
 type Field = FieldWithOptions | FieldWithoutOptions;
 
 type FieldSetProps = {
-  legend: string;
+  label: string;
   name: string;
+  description: string;
   fields: Field[];
 };
 
-export function FieldSetField({ legend, name, fields }: FieldSetProps) {
+export function FieldSetField({
+  label,
+  name,
+  fields,
+  description,
+}: FieldSetProps) {
   return (
     <fieldset
       className={cn(
         'border-1 border-input p-4 rounded-xl',
         `RemoteFlows__FieldSetField__${name}`,
       )}
     >
-      <legend className="text-sm font-semibold px-2">{legend}</legend>
```

Comment:
Previously this would be `undefined`. It now, updates the prop from `legend` to `label` as this is what we get from `json-schema-form`.
<img width="723" alt="image" src="https://github.com/user-attachments/assets/230a94d0-a6d5-4d31-a55e-c52d851a6ad9" />

Reply:
It also renders the `description`